moveit_ros/moveit_servo/include/moveit_servo/collision_check.h

```diff
@@ -86,8 +86,8 @@ class CollisionCheck
   // Pointer to the ROS node
   const std::shared_ptr<rclcpp::Node> node_;
 
-  // Parameters from yaml
-  const ServoParameters::SharedConstPtr parameters_;
+  // The servo parameters
+  servo::Params servo_params_;
```

Comment:
Nitpick: it's pretty obvious what these are from the name I think, the comment isn't needed.

```suggestion
  servo::Params servo_params_;
```